changelog.json

```
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.5.0": [
+      "[New] Add, push, and view git tags in Desktop - #9424",
+      "[Added] Offer option to choose file from one branch or the other when resolving merge conflicts - #9702",
+      "[Added] Add context menu for highlighted text in diff view - #5100. Thanks @HashimotoYT!",
+      "[Fixed] Retain default branch option in \"Create Branch\" dialog when user focuses away from Desktop - #9611",
+      "[Improved] Clean up copy for onboarding sign-in flow - #9715",
+      "[Improved] Emphasize sign in via browser option in Welcome flow - #9663",
```

Just to provide a bit more context here
```suggestion
      "[Improved] Emphasize signing in to GitHub via browser due to username/password deprecation - #9663",
```

---

hmmm i don't know if that deprecation is publicly announced yet @billygriffin. or has it?

---

@outofambit It has been! https://developer.github.com/changes/2020-02-14-deprecating-oauth-auth-endpoint/